Fitness/Controllers/HomeController.swift

```diff
@@ -8,12 +8,19 @@
 import UIKit
 import SnapKit
 
+enum SectionType {
+    case allGyms
+    case todaysClasses
+    case lookingFor
+}
+
 class HomeController: UIViewController, UITableViewDataSource, UITableViewDelegate {
     
     // MARK: - INITIALIZATION
     var tableView: UITableView!
     var statusBarBackgroundColor: UIView!
     
+    var sections = [SectionType]()
```

Comment:
you should be doing `var sections: [SectionType] = []`. same for the others after this as well